rust_src/remacs-sys/build.rs

```diff
@@ -196,7 +197,36 @@ fn generate_globals() {
     }
 }
 
+fn generate_module_code() {
+    let mut builder = bindgen::Builder::default()
+        .rust_target(bindgen::RustTarget::Nightly)
+        .generate_comments(true)
+        .header("../../src/emacs-module.h")
+        .rustified_enum("emacs_funcall_exit")
+        .ctypes_prefix("::libc");
+
+    if cfg!(windows) {
+        builder = builder
+            .clang_arg("-fms-extensions")
+            .clang_arg("-fms-compatibility")
+            .clang_arg("-fms-compatibility-version=19")
+            .clang_arg("-D_MSC_VER=1900")
+            .clang_arg("-D_M_X64")
+    }
+
+    let bindings = builder
+        .rustfmt_bindings(true)
+        .generate()
+        .expect("Unable to generate bindings");
+
+    let source = bindings.to_string();
+    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap()).join("modules.rs");
+    let file = File::create(out_path);
+    file.unwrap().write_all(source.as_bytes()).unwrap();
```

Comment:
Checkout the error handling I added to the top level build.rs. If this function returned a `Result` with `io::Error` you could use `?` handling to clarify this logic and avoid `unwrap` failures.